pkg/apis/config/feature_flags.go

```diff
@@ -88,6 +90,8 @@ const (
 	DefaultSetSecurityContext = false
 	// DefaultCoschedule is the default value for coschedule
 	DefaultCoschedule = CoscheduleWorkspaces
+	// EnableCancelUsingEntrypoint is the flag used to enable cancelling a pod using the entrypoint
+	EnableCancelUsingEntrypoint = "enable-cancel-using-entrypoint"
```

Comment:
Could you add this feature flag here please? https://github.com/tektoncd/pipeline/blob/main/config/config-feature-flags.yaml

Reply:
added